Directory.Build.targets

```diff
@@ -68,4 +70,70 @@
   <Import Project="eng\targets\FSharp.Common.targets"  Condition="'$(MSBuildProjectExtension)' == '.fsproj'" />
   <Import Project="eng\targets\Wix.Common.targets"  Condition="'$(MSBuildProjectExtension)' == '.wixproj'" />
   <Import Project="eng\targets\Npm.Common.targets"  Condition="'$(MSBuildProjectExtension)' == '.npmproj'" />
+
+  <ItemGroup>
+    <PackageReference Include="xunit.runner.console" Version="2.4.1" Condition="'$(TargetFrameworkIdentifier)' == '.NETFramework'" />
+  </ItemGroup>
+
+  <Target Name="CreateHelixPayload" Returns="@(HelixPayload)">
+    <ItemGroup>
+      <_TargetFrameworks Remove="@(_TargetFrameworks)" />
+      <_TargetFrameworks Include="$(TargetFrameworks);$(TargetFramework)" />
+    </ItemGroup>
+
+    <MSBuild Projects="$(MSBuildProjectFullPath)"
+             Targets="_CreateHelixPayloadInner"
```

Comment:
BuildInParallel?

Reply:
Ok now I remember why build in parallel is false, some publish directories (for example IIS/Common fail when publishing in parallel: 

```
CSC : error CS2012: Cannot open 'C:\Github\AspNetCore\src\Servers\IIS\IIS\test\Common.Tests\obj\Debug\netcoreapp3.0\Com
mon.Tests.dll' for writing -- 'The process cannot access the file 'C:\Github\AspNetCore\src\Servers\IIS\IIS\test\Common
.Tests\obj\Debug\netcoreapp3.0\Common.Tests.dll' because it is being used by another process.' [C:\Github\AspNetCore\sr
c\Servers\IIS\IIS\test\Common.Tests\Common.Tests.csproj]
```